Update taxonomy-api.md (#101)
amplitude/amplitude-dev-center: docs/analytics/apis/taxonomy-api.md

`GET https://amplitude.com/api/2/taxonomy/event`

!!!note

>     Events that are hidden (i.e. that have a visibility of anything other than "Visible") will not appear in the response.

Reviewer: ```suggestion
    Hidden events, those that have a visibility other than "Visible", don't appear in the response.
```

>     By default, deleted events will also not be included, but the `showDeleted` query parameter can be optionally added to the endpoint to include them, e.g.:

Reviewer: @ZahraTee Are there other parameters for this endpoint?

Author: Not that I know of.
